File: Languages/Ruby/ruby_study_5/page8/food.rb
require "./menu"

class Food < Menu
  attr_accessor :calorie
  
  # Definisikan method `initialize`
  def initialize(name:, price:, calorie:)
    self.name = name
    self.price = price
    self.calorie = calorie
  end
  
  def info
    return "#{self.name} $#{self.price} (#{self.calorie}kcal)"
  end
  
  def calorie_info
    return "#{self.name} mengandung #{self.calorie}kcal"
  end
end

File: Languages/Python/python_study_5/page6/food.py
from menu_item import MenuItem

class Food(MenuItem):
    def info(self):
        return self.name + ': $' + str(self.price) + ' (' + str(self.calorie_count) + 'kkal)'
    
    def calorie_info(self):
        print('kkal: ' + str(self.calorie_count))